src/store/messagesStore.js

```diff
@@ -1306,7 +1306,8 @@ const actions = {
 		for (const key in message.messageParameters) {
 			if (key.startsWith('mention')) {
 				const mention = message.messageParameters[key]
-				message.message = message.message.replace(`{${key}}`, `@${mention.name}`)
+				const mentionString = key.includes('mention-call') ? `**${mention.name}**` : `@"${mention.id}"`
```

Comment:
Not sure it causes issues somewhere, but usually we only quote when a space or slash is contained in the id

Reply:
It also works for ids that do not contain spaces of slashes, maybe a specific use case I am not aware of?

Reply:
https://github.com/nextcloud-libraries/nextcloud-vue/blob/master/src/components/NcRichContenteditable/NcMentionBubble.vue#L85-L89

Maybe it's only on NcRichContenteditable